libp2p/standard_setup.nim

```diff
@@ -27,7 +27,11 @@ proc newStandardSwitch*(privKey = none(PrivateKey),
                         transportFlags: set[ServerFlags] = {},
                         rng = newRng(),
                         inTimeout: Duration = 5.minutes,
-                        outTimeout: Duration = 5.minutes): Switch =
+                        outTimeout: Duration = 5.minutes,
+                        maxConnections = MaxConnections,
```

Comment:
regarding backwards compatibility, this for example would require a major version bump since it's an incompatible behavioural change - a backwards-compatible way would be to default to unlimited, but this perhaps isn't a good "default"

Reply:
> major version bump since it's an incompatible behavioural change

Sure, that's why we use versions in the first place, to signal changes, not how good we feel about it :)